Changed the title use use SiteIcon, etc if defined

bengillies opened the pull request:
The title now defaults to the same as the TiddlyWiki representation (the
defaultSiteIcon and the tagline of "a TiddlySpace"). If the user has a
SiteIcon of their own, or a SiteSubtitle, then it will use that instead.

bengillies committed:
Changed the title use use SiteIcon, etc if defined

The title now defaults to the same as the TiddlyWiki representation (the
defaultSiteIcon and the tagline of "a TiddlySpace"). If the user has a
SiteIcon of their own, or a SiteSubtitle, then it will use that instead.

cdent commented on src/system-applications/apps.tid:
```diff
@@ -16,12 +17,12 @@ tags: excludeLists excludeSearch
 				<div id="title-subtitle">
 					<h1 class="spaceaddress">
-						<span>SPACENAME</span>.tiddlyspace.com
+						<span class="spaceName"></span>.tiddlyspace.com
```
This can't reference tiddlyspace.com. That's not portable to other hosts. The space's entire hostname needs to be calculated.

bengillies replied:
Good point

cdent commented on src/system-applications/apps.tid:
```diff
@@ -9,6 +9,7 @@ tags: excludeLists excludeSearch
 	<title>TiddlySpace Apps</title>
 	<link rel="stylesheet" href="http://tiddlyspace.com/bags/common/tiddlers/reset.css" />
 	<link rel="stylesheet" href="appspage.css" />
+	<link rel="stylesheet" href="/bags/common/tiddlers/backstage.css" />
```
backstage.js loads backstage.css itself, yes, so this isn't needed is it?

bengillies replied:
Didn't know that. I'll update

bengillies committed:
Made hostName in title dynamic and removed backstage.css

tiddlyspace.com should not be hardcoded into the title, as it can be
installed on any domain.

backstage.css is auto applied by backstage.js, so it should not be
included statically
